client/src/components/RatingsAndReviews/WritingReview/SetStarRating.scss

```diff
@@ -0,0 +1,9 @@
+
+.star_radio[type="radio"] {
+  display: none;
+}
+
+.star {
+  cursor: pointer;
+  transition: color 200ms;
+}
```

Comment:
Could you please add a new line at the end of the file? Just a tiny best practice thing.